### PR TITLE
[feature request] add cpu burst strategy

### LINKED ISSUE #52: [feature request] add cpu burst strategy
**What would you like to be added**:

**Why is this needed**:

### PR DESCRIPTION
**Pull request by stormgbs**
### Ⅰ. Describe what this PR does

fixes #52

### CODE REVIEW

File: apis/extension/slo-config.go [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+package extension
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	slov1alpha1 "github.com/koordinator-sh/koordinator/apis/slo/v1alpha1"
+)
+
+type NodeCPUBurstCfg struct {
```
Comment (jasonliu747) [inline apis/extension/slo-config.go:25]: we define these configs in `pkg/slo-controller/config/config.go`

**Review by jasonliu747 — CHANGES_REQUESTED** [review]

**Review by jasonliu747 — COMMENTED** [review]
lgtm overall, others PTAL
/cc @eahydra @zwzhang0107 @saintube

**Review by saintube — COMMENTED** [review]
/lgtm

File: pkg/slo-controller/nodeslo/nodeslo_controller_test.go [hunk 2]
```diff
@@ -222,6 +226,7 @@ func TestNodeSLOReconciler_Reconcile(t *testing.T) {
```
Comment (jasonliu747) [inline pkg/slo-controller/nodeslo/nodeslo_controller_test.go:226]:
```
testingCPUBurstStrategy := util.DefaultCPUBurstStrategy()
testingCPUBurstStrategy.CFSQuotaBurstPeriodSeconds = util.Int64Ptr(60)
```

```
nodeSLOSpec := &slov1alpha1.NodeSLOSpec{
	ResourceUsedThresholdWithBE: testingResourceThresholdStrategy,
	CPUBurstStrategy:            testingCPUBurstStrategy,
}
```

File: pkg/util/config.go [hunk 3]
```diff
@@ -37,3 +37,19 @@ func DefaultResourceThresholdStrategy() *slov1alpha1.ResourceThresholdStrategy {
 		MemoryEvictThresholdPercent: pointer.Int64Ptr(70),
 	}
 }
+
+func DefaultCPUBurstConfig() slov1alpha1.CPUBurstConfig {
```
Comment (jasonliu747) [inline pkg/util/config.go:41]: can you add some explanations for these default values?

File: pkg/slo-controller/nodeslo/nodeslo_controller_test.go (hunk 2 above)
Comment (zwzhang0107) [reply pkg/slo-controller/nodeslo/nodeslo_controller_test.go:226]: +1，also add 60 to testingConfigMap.

**Comment by jasonliu747** [comment]
@stormgbs Also, please rename your PR title...

**Pull request closed without merging**